pkg/stub/psmdb.go

```diff
@@ -173,22 +173,16 @@ func (h *Handler) newStatefulSetVolumes(m *v1alpha1.PerconaServerMongoDB) ([]cor
 
 	if h.hasBackupsEnabled(m) {
 		agentVolumes, err := h.backups.NewAgentVolumes()
-		if err != nil {
-			return nil, err
+		if err == nil {
+			volumes = append(volumes, agentVolumes...)
```

Comment:
I meant take the NewAgentVolumes() out of newStatefulSetVolumes() - to call and handle it separately. 
Omitting the error could bring more problem. Now if NewAgentVolumes() fails we'll never see any message in logs... 

So if it takes a lot of time to split this method, just bring the error trowing back. It's more informative and easier to handle if it happens.

Thanks!

Reply:
Ok, I have reverted this change. We can refactor it later